From d3182c984a563b654260acf56ba9fae420b906fe Mon Sep 17 00:00:00 2001
From: Steve deRosier <steve.derosier@lairdtech.com>
Date: Mon, 4 Jan 2016 09:15:30 -0800
Subject: [PATCH] laird: dts: wb50n: implment osc-bypass

This change switches the slow-clock to osc-bypass mode for the new
B1 boards that have an external osc instead of a raw crystal on the
32kHz clock. While this makes the new B1 boards work correctly, it
is incompatable with B0 and older boards and effecively makes them
useless.

Bug: 8785
---
 arch/arm/boot/dts/wb50n.dts | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/wb50n.dts b/arch/arm/boot/dts/wb50n.dts
index f062f39..d6dda78 100644
--- a/arch/arm/boot/dts/wb50n.dts
+++ b/arch/arm/boot/dts/wb50n.dts
@@ -151,10 +151,11 @@
 				status = "okay";
 			};
 
-
-
-
-
+			sckc@fffffe50 {
+				slow_osc: slow_osc {
+					atmel,osc-bypass;
+				};
+			};
 		};
 
 		nand0: nand@60000000 {
-- 
1.9.1

